### PR TITLE
gradewindow connected to database

### PR DESCRIPTION
**Pull request by nehadhyani**
gradewindow connected to database
session variables used

### CODE REVIEW

File: Codes/edit.php [hunk 1]
```diff
@@ -87,17 +67,6 @@
...
-        if (isset($_GET['course']))
-        {
-          $cd = $_GET['course'];
-        }
-        else  
-        {
-          header('Location: dashboard.php?error=ERROR ');
-          exit();
-        }
-        ?>
```
Comment (Anupam-Panwar) [inline Codes/edit.php:100]: any feasible explanation for removing this part

File: Codes/edit.php [hunk 2]
```diff
@@ -186,60 +157,31 @@
...
-                            <td><input class="gradewin"></input></td>
-                            <td><input class="gradewin"></input></td>
-                            <td><input class="gradewin"></input></td>
-                            <td><input class="gradewin"></input></td>
-                            <td><input class="gradewin"></input></td>
-                            <td><input class="gradewin"></input></td>
-                            <td><input class="gradewin"></input></td>
-                            <td><input class="gradewin"></input></td>
-                            <td><input class="gradewin"></input></td>                       
+                            <?php } ?>
```
Comment (Anupam-Panwar) [inline Codes/edit.php]: think about any feasible way to link corresponding grade with cut off

File: Codes/edit.php (hunk 1 above)
Comment (nehadhyani) [reply Codes/edit.php:100]: This part is repeated here. --line 28

**Comment by nehadhyani** [comment]
-insert to coursetable redirection done
-print, edit, insert - first session then course checked
-edit, print - cutoff and total students according to the grades

**Pull request merged**